integration/inspektor-gadget/integration_test.go

```diff
@@ -764,10 +765,27 @@ func TestOomkill(t *testing.T) {
 	t.Parallel()
 
 	oomkillCmd := &Command{
-		Name:           "StarOomkilGadget",
-		Cmd:            fmt.Sprintf("$KUBECTL_GADGET trace oomkill -n %s", ns),
-		ExpectedRegexp: `\d+\s+tail`,
-		StartAndStop:   true,
+		Name:         "StartOomkilGadget",
+		Cmd:          fmt.Sprintf("$KUBECTL_GADGET trace oomkill -n %s -o json", ns),
+		StartAndStop: true,
+		ExpectedOutputFn: func(output string) error {
+			expectedEntry := &oomkillTypes.Event{
+				Event:         BuildBaseEvent(ns),
+				KilledComm:    "tail",
+				TriggeredComm: "tail",
```

Comment:
> The triggered command is most likely tail itself but technically it is not guaranteed. Should i modify the exptectedEntry?
>
@burak-ok, I think @eiffel-fl could help you with this question.

Reply:
> I think we should try to stick with the regex when converting tests to JSON.
>
I don't follow you. In which way?

Reply:
I think the new implemenation Burak just pushed is the correct way to go. What are you proposing?

Reply:
The previous test with `regex` tested on only one occurence of `tail`